build/common.mk

```diff
@@ -106,17 +104,52 @@ endif
 PULUMI_BIN          := $(PULUMI_ROOT)/bin
 PULUMI_NODE_MODULES := $(PULUMI_ROOT)/node_modules
 
-.PHONY: default all ensure only_build only_test only_test_fast build lint install test_fast test_all core
+.PHONY: default all ensure only_build only_test build lint install test_fast test_all core
```

Comment:
why did only_test_fast go away?

Reply:
I don't know, this came from another repo